design/ordinal-crm.Rmd

```diff
@@ -0,0 +1,453 @@
+      &= \int_{\mathbb R} \tfrac{1}{\sigma_2} \tfrac{\varphi(\tfrac{x_2 - \mu_2}{\sigma_2})}{\Phi(\tfrac{x_1 - \mu_2}{\sigma_2})}  I(x_2 < x_1) \tfrac{1}{\sigma_1} \varphi(\tfrac{x_1 - \mu_1}{\sigma_1}) \, dx_1 \\
+      &= \tfrac{1}{\sigma_1 \sigma_2} \varphi(\tfrac{x_2 - \mu_2}{\sigma_2}) \int_{x_2}^{\infty} \tfrac{\varphi(\tfrac{x_1 - \mu_1}{\sigma_1})}{\Phi(\tfrac{x_1 - \mu_2}{\sigma_2})} \, dx_1 \\
+      &= ...
+      \end{aligned}
+    $$ for $x_2 \in \mathbb R$, where $\varphi$ is the probability density function of the standard
+    normal distribution and $\Phi$ is its cumulative distribution function. Below is the example
+    shape of $f_{\alpha_2}$ function.
+
+```{r f_alpha2, eval=TRUE, fig.height = 3, fig.width = 5, fig.align = "center"}
+    f <- function(x2, mu1 = 8, mu2 = 5, sd1 = 2, sd2 = 2) {
+      phi <- dnorm
+      Phi <- pnorm
+      to_integrate <- function(x1, mu1, mu2, sd1, sd2) {
+        phi((x1 - mu1) / sd1) / Phi((x1 - mu2) / sd2)
+      }
+      intgrl <- integrate(
+        to_integrate,
+        lower = x2, upper = Inf,
+        mu1 = mu1, mu2 = mu2, sd1 = sd1, sd2 = sd2
+      )
+      1 / (sd1 * sd2) * phi((x2 - mu2) / sd2) * intgrl$value
+    }
+    f <- Vectorize(f)
+    
+    ggplot() +
+      geom_function(fun = dnorm, colour = "black", args = list(mean = 5, sd = 2)) +
+      geom_function(fun = f, colour = "red") +
+      scale_x_continuous(breaks = seq(-5, 15, 2), limits = c(-5, 15)) +
+      xlab("x2") +
+      ylab("PDF") +
+      ggtitle("PDF of: N(5, sd = 2) [black] and alpha2 (uncond.) [red]")
+    
+    optimize(f, interval = c(1, 7), maximum = TRUE)
+```
+
+3.  In JAGS, we can specify $\alpha_j,\, j = 2, \ldots, c - 1$ with
+
+    ```{=tex}
+    \begin{center}
+      \tt alpha[j] $\sim$ dnorm(meanAlpha[j], precAlpha[j]) T(, alpha[j-1])
+    \end{center}
+    ```
+    However, I think we need to make sure the sample that is being obtained by this is indeed from
+    an unconditional distribution of $\alpha_j$, $j = 2, \ldots, c-1$.
+
+### Multivariate Truncated Normal Distribution
+
+We specify this prior for model \eqref{model}.
+
+This approach allows a user for a full control of var-cov structure of random vector
+$\boldsymbol \alpha$. Following [15], we specify distribution of $\boldsymbol \alpha$ as
+multivariate truncated normal distribution. Consider first
+$\boldsymbol \alpha^* \sim N_{c-1}(\boldsymbol \mu, \Sigma)$. Now, let $\boldsymbol \alpha$ be
+truncation of $\boldsymbol \alpha^*$ above $\boldsymbol k \in \mathbb R^{c-1}$. $\boldsymbol \alpha$
+has a $(c-1)$-variate truncated normal distribution given by $$
+f_{\boldsymbol \alpha}(\boldsymbol x; \boldsymbol \mu, \Sigma, \boldsymbol k) =
+\frac{exp\{-\tfrac{1}{2}(\boldsymbol x - \boldsymbol \mu)^T \Sigma (\boldsymbol x - \boldsymbol \mu)\}}
+{\idotsint_{R^{c-1}_{\leq \boldsymbol k}} exp\{-\tfrac{1}{2}(\boldsymbol x - \boldsymbol \mu)^T \Sigma (\boldsymbol x - \boldsymbol \mu)\} \,dx_1 \dots dx_{c-1}}
+$$ where $$
+\begin{aligned}
```

Comment:
```suggestion
has a $(c-1)$-variate truncated normal distribution given by 
\[
f_{\boldsymbol \alpha}(\boldsymbol x; \boldsymbol \mu, \Sigma, \boldsymbol k) =
\frac{exp\{-\tfrac{1}{2}(\boldsymbol x - \boldsymbol \mu)^T \Sigma (\boldsymbol x - \boldsymbol \mu)\}}
{\idotsint_{R^{c-1}_{\leq \boldsymbol k}} exp\{-\tfrac{1}{2}(\boldsymbol x - \boldsymbol \mu)^T \Sigma (\boldsymbol x - \boldsymbol \mu)\} \,dx_1 \dots dx_{c-1}}
\] 
where 
\[
\begin{aligned}
```
etc, try to use `\[` instead of `$$` generally

Reply:
another Rstudio editing issue, i.e. even if you use `\[`, then it is auto-corrected back to `$$`.